src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -1079,6 +1081,24 @@ public static String prefixName(String name) {
     return PDAO_PREFIX + name;
   }
 
+  private Schema buildRowMetadataSchema() {
+    List<Field> fieldList = new ArrayList<>();
+    fieldList.add(Field.of(PDAO_ROW_ID_COLUMN, LegacySQLTypeName.STRING));
+    fieldList.add(
+        Field.newBuilder("ingested_by", LegacySQLTypeName.STRING)
```

Comment:
I'd expect to see a named constant for these field names so they can be referenced by other code.